daprdocs/content/en/reference/api/pubsub_api.md

```diff
@@ -92,13 +93,24 @@ Example:
   {
     "pubsubname": "pubsub",
     "topic": "newOrder",
-    "route": "/orders"
+    "route": "/orders",
+    "metadata": {
+      "rawPayload": "false",
```

Comment:
Is there any scenario where someone would explicitly set rawPayload to false? Wouldn't the primary behavior be to just omit the value?

Asking because I don't want to show an example of something that's not the primary/recommended behavior.

Reply:
@artursouza  - Can you address the question from @AaronCrawfis ?

Reply:
I've added something, so users can see how metadata can be returned. Yes, `false` is the default. I've changed it to true - but that is not the recommended value since there are many disadvantages of setting it to false and the publisher must also override this.